components/demo/demo-snippet.js

```diff
@@ -201,10 +231,14 @@ class DemoSnippet extends LitElement {
 		this._applyAttr('dir', this._dir, true);
 	}
 
-	_handleFullscreenChange(e) {
+	async _handleFullscreenChange(e) {
 		this._fullscreen = e.target.on;
+		this._settingsPeek = this._fullscreen;
 		const event = new CustomEvent('d2l-demo-snippet-fullscreen-toggle', { bubbles: true, composed: true });
 		this.dispatchEvent(event);
+		await this.update;
```

Comment:
This isn't technically necessary but seems like good form.

Reply:
Is this meant to be `await this.updateComplete;`?

Also, if I understand correctly, the purpose of `_settingsPeek` is to just show the dropdown opener briefly in the top-right corner upon toggling to fullscreen, is that correct?

Reply:
Whoops, yes on both counts.